sootup.java.core/src/main/java/sootup/java/core/JavaIdentifierFactory.java

```diff
@@ -56,6 +56,32 @@ public class JavaIdentifierFactory implements IdentifierFactory {
 
   @Nonnull private static final JavaIdentifierFactory INSTANCE = new JavaIdentifierFactory();
 
+  public boolean isStaticInitializerSignature(@Nonnull MethodSubSignature methodSubSignature) {
+    return methodSubSignature.getName().equals("<clinit>");
```

Comment:
there is a static String for this JavaSootMethod.STATIC_INITIALIZER_NAME

Reply:
moved it to the javaIDF